rpc/src/v1/types/action.rs

```diff
@@ -67,6 +67,15 @@ pub enum Action {
         approvals: Vec<Signature>,
     },
     #[serde(rename_all = "camelCase")]
+    IncreaseAssetSupply {
+        network_id: NetworkId,
+        shard_id: ShardId,
+        asset_type: H160,
+        output: Box<AssetMintOutput>,
```

Comment:
I think this transaction needs a new output type.
The supply of `AssetMintOutput` is Option, but the supply of `IncreaseAssetSupply` should be not.

Reply:
I discussed it with @kseo .
Let's change the supply of `AssetMintOutput` to `u64` instead of `Option<u64>`.
And make the sdk fills the supply to the maximum of `u64` if it's null.